.github/workflows/ci-cd.yml

```diff
@@ -54,7 +54,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        project: ["Sailfish", "Sailfish.TestAdapter", "Sailfish.Analyzers"]
+        project: ["Sailfish"]
```

Comment:
Previously each matrix would build the same three projects, which meant we build every project three times and uploaded the same artifact 3 times.

This does not fly under v4 of upload artifact, so we now just do it once. I am sure it doesn't need to be a matrix job, but I don't really know what is involved in not making a matrix job. I am sure it is just a matter of removing matrix